rmd/Quests.json

```diff
@@ -2146,13 +2146,13 @@
   },
   "210151": {
     "OriginalText": "鉱物を回収しろ",
-    "Text": "Collect the minerals",
+    "Text": "Collect the Minerals",
     "Enabled": true
   },
   "210152": {
     "OriginalText": "拠点を守れ   拠点耐久",
-    "Text": "",
-    "Enabled": false
+    "Text": "Protect Base (Durability)",
```

Comment:
I'm not sure of the context of how this will look in game due to how it's separated out the way it is, so keep an eye out for it. I assume what it's doing is putting the durability numerics below this line or something, but I'm not sure.